metadata/overlays/ibm-storage-config/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../../base
+images:
+  - name: mysql
+    newTag: "5.6"
```

Comment:
Same here. Is the DB version sufficient?

Reply:
The metadata-db is using mysql to do basic database operations similar to pipeline's mysql. Therefore, we aligned with pipeline's mysql 5.6 and it works smoothly on our environment.